Rakefile

```diff
@@ -24,57 +24,88 @@ end
-  end
-  t.add_flag do |flag|
-    flag.name = '--hosts'
-    flag.default = 'acceptance/hosts.cfg'
-    flag.message = 'The configuration file that Beaker will use'
-    flag.override_env = 'BEAKER_HOSTS'
-  end
-  t.add_flag do |flag|
-    flag.name = '--preserve-hosts'
-    flag.default = 'onfail'
-    flag.message = 'The beaker setting to preserve a provisioned host'
-    flag.override_env = 'BEAKER_PRESERVE_HOSTS'
-  end
-  t.add_flag do |flag|
-    flag.name = '--keyfile'
-    flag.default ="#{ENV['HOME']}/.ssh/id_rsa-acceptance"
-    flag.message = 'The SSH key used to access a SUT'
-    flag.override_env = 'BEAKER_KEYFILE'
-  end
-  t.add_flag do |flag|
-    flag.name = '--load-path'
-    flag.default = 'acceptance/lib'
-    flag.message = 'The load path Beaker will use'
-    flag.override_env = "BEAKER_LOAD_PATH"
-  end
-  t.add_flag do |flag|
-    flag.name = '--pre-suite'
-    flag.default = 'acceptance/pre-suite'
-    flag.message = 'THe path to a directory containing pre-suites'
-    flag.override_env = "BEAKER_PRE_SUITE"
-  end
-  t.add_flag do |flag|
-    flag.name = '--tests'
-    flag.default = 'acceptance/tests'
-    flag.message = 'The path to the tests you want beaker to run'
-    flag.override_env = 'BEAKER_TESTS'
+  #t.add_flag do |flag|
+    #flag.name = '--log-level'
+    #flag.default ="verbose"
+    #flag.message = 'beaker log-level'
+    #flag.override_env = 'BEAKER_LOG_LEVEL'
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--hosts'
+    #flag.default = 'acceptance/hosts.cfg'
+    #flag.message = 'The configuration file that Beaker will use'
+    #flag.override_env = 'BEAKER_HOSTS'
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--preserve-hosts'
+    #flag.default = 'onfail'
+    #flag.message = 'The beaker setting to preserve a provisioned host'
+    #flag.override_env = 'BEAKER_PRESERVE_HOSTS'
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--keyfile'
+    #flag.default ="#{ENV['HOME']}/.ssh/id_rsa-acceptance"
+    #flag.message = 'The SSH key used to access a SUT'
+    #flag.override_env = 'BEAKER_KEYFILE'
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--load-path'
+    #flag.default = 'acceptance/lib'
+    #flag.message = 'The load path Beaker will use'
+    #flag.override_env = "BEAKER_LOAD_PATH"
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--pre-suite'
+    #flag.default = 'acceptance/pre-suite'
+    #flag.message = 'THe path to a directory containing pre-suites'
+    #flag.override_env = "BEAKER_PRE_SUITE"
+  #end
+  #t.add_flag do |flag|
+    #flag.name = '--tests'
+    #flag.default = 'acceptance/tests'
+    #flag.message = 'The path to the tests you want beaker to run'
+    #flag.override_env = 'BEAKER_TESTS'
+  #end
+
+  t.add_command({:name => 'beaker --log-level verbose --hosts acceptance/hosts.cfg --preserve-hosts --keyfile ~/.ssh/id_rsa-acceptance --load-path acceptance/lib/ --pre-suite acceptance/pre-suite --tests acceptance/tests/', :override_env => 'BEAKER_EXECUTABLE'})
```

Comment:
that is using the new API.  just not in the best way.  add_flag is totally hosed right now, i forget why i had to do it that way, i think due to moving all the files around and adding command_collections.